data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/InvalidCloudEventInterceptor.java

```diff
@@ -61,6 +61,8 @@
 
   public static final String TYPE = "dev.knative.kafka.event";
 
+  private static final String KEY_EXTENSION = "key";
```

Comment:
can we - somewhere, get a little comment/description on the relationship between `PARTITION_KEY_EXTENSION` and `KEY_EXTENSION` ?

Reply:
The only reason to use `key` is for backward compatibility with the exiting source in `eventing-kafka`.

I'd propose to deprecate that field if possible.

Reply:
> So, I'd deprecate key and the alternative will be partitionkey.

sounds good, @pierDipi